xmltest.cpp

```diff
@@ -63,6 +63,15 @@ bool XMLTest (const char* testString, const char* expected, const char* found, b
 	return pass;
 }
 
+bool XMLTest(const char* testString, XMLError expected, XMLError found, bool echo = true, bool extraNL = false)

```

Comment:
From my commit comment: Updated tests to print ErrorID and bool values as strings.
I found the tests printed lots of 0/1 for false/true and similarly numbers for XMLError values.
I found this made it harder to find which test was failing.
It also meant I had to manually convert numbers back to XMLError identifiers.
With this change bool and XMLError results are printed in a clearer format.